code/controllers/subsystem/shuttles.dm

```diff
@@ -68,6 +73,65 @@ var/datum/subsystem/shuttle/SSshuttle
 			P.check()
 			continue
 		mobile.Remove(thing)
+	if(prob(1) && prob(10))
+		fluctuate()
+	if(prob(1) && prob(10))
+		var/datum/shipping/S = pick(shipping_datums)
+		if(S.allow_import) // no more shortages of syndicate documents
+			var/event = pick(1,2)
+			var/percentage = rand(25,75)
+			switch(event)
+				if(1)
+					priority_announce("Crew, there's a galaxy wide shortage of [S.name]. The Space Trade Commission is reporting prices to be going up by [percentage]%, and we recommend exporting.")
+					var/temp = (percentage / 100) * S.value
+					var/temp_amt = round((percentage / 100) * S.amount_on_market)
+					S.amount_on_market -= temp_amt
+					S.add_value(temp)
+				if(2)
+					priority_announce("Crew, there's a galaxy wide surplus of [S.name]. The Space Trade Commission is reporting prices to be going down by [percentage]%, and we recommend importing.")
```

Comment:
Why a station-wide announcement instead of something limited to cargo? Only cargo is affected by this, not all the station.

Also it looks like it would better be a random event.